aten/src/ATen/native/transformers/cuda/flash_attn/fmha_api.cpp

```diff
@@ -128,8 +128,10 @@ mha_fwd(const at::Tensor &q,         // total_q x num_heads x head_size, total_q
         const float softmax_scale,
         const bool zero_tensors,
         const bool is_causal,
-        const bool return_softmax,
         c10::optional<at::Generator> gen_) {
+    // return_softmax is an a parameter for flash attention
```

Comment:
return_softmax is a parameter for flash attention but for the in core api though we are removing this parameter.